documentation/modules/ROOT/pages/modules/monitoring/r_kafka-connect-jmx-environment-variables-docker.adoc

```diff
@@ -0,0 +1,40 @@
+[id="kafka-connect-jmx-environment-variables-docker"]
+= Kafka Connect JMX environment variables in Docker
+
+The `debezium/connect` image recognizes the following JMX-related environment variables:
```

Comment:
@bhardesty, to be sure, are you replacing these parts with something else for downstream? The debezium/connect and debezium/kafka images are upstream deliverables, but shouldn't be referenced in downstream docs.

Reply:
@gunnarmorling Since each of these sections is a separate file, I'll only include the downstream ones in the downstream doc (i.e. the local installation env variables).

Also, I forgot to mention that this commit should probably also be cherry-picked to the 1.0 release branch. Do I need to create a separate PR for that?

Reply:
> I'll only include the downstream ones in the downstream doc

@bhardesty, ok, so this means you got alternative filels downstream which talk about JMX with the AMQ Streams then?

>  Do I need to create a separate PR for that?

Yes, please. I realize backporting of your recent doc changes has been handled a bit suboptimal at this point. We'll need to think about a better process for the next release. I'll do a pass with @Naros to make sure nothing got forgotten.

Reply:
@gunnarmorling No, I just meant that I will only include the sections that are relevant for downstream. That would be the sections about JMX for local installations.

Reply:
Ok, so there'd be no monitoring docs for AMQ Streams on OpenShift then?

Reply:
@gunnarmorling For the downstream, I can add a short module that directs readers to the AMQ Streams on OpenShift monitoring doc: 

https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html-single/using_amq_streams_on_openshift/index#assembly-metrics-setup-str

Reply:
Ok, thanks!